scrapers/utils.py

```diff
@@ -111,7 +112,7 @@ def validate_and_get_download_link(base_url, download_link, check_content_pdf=Tr
         response = requests.head(absolute_url)
         content_type = response.headers.get('Content-Type', '')
         is_pdf = 'application/pdf' in content_type
-        if not is_pdf:
+        if not is_pdf and download_link[-4:] != '.pdf':
```

Comment:
This is probably better

```suggestion
        if not is_pdf and download_link.endswith('.pdf'):
```